cfgov/unprocessed/js/routes/consumer-tools/financial-well-being/index.js

```diff
@@ -0,0 +1,85 @@
+/* eslint no-global-assign: "off" */
```

Comment:
I don't see how this is needed. I removed it and ran the linter and didn't see a difference.

Reply:
I think it was copied from the original JS on the Makeathon prototype, but if it can go, I'm cool with that. The project needs a more stringent JS review at a later date, anyway, though.